### PR TITLE
./mode is shorthand for ./mode list, likewise ./vote is shorthand for ./vote list

### PR DESCRIPTION
**Pull request by dentmaged**
image

`/mode` and `/modes` without any arguments will run `/mode list`. Running a child command that doesn't exist will send the parent command help (as seen with `/mode a`).

### CODE REVIEW

**Comment by Pablete1234** [comment]
Make it so `mode` and `modes` are just aliases for the same thing, and the default if no subcommand is passed is showing the list

File: core/src/main/java/tc/oc/pgm/command/ModesCommand.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import app.ashcon.intake.CommandException;
+import app.ashcon.intake.parametric.annotation.Default;
+import tc.oc.pgm.api.match.Match;
+import tc.oc.pgm.util.Audience;
+
+public final class ModesCommand {
+
+  @Command(
+      aliases = {"modes"},
+      desc = "List all objectivs modes",
```
Comment (CoWinkKeyDinkInc) [inline core/src/main/java/tc/oc/pgm/command/ModesCommand.java]: typo

**Comment by CoWinkKeyDinkInc** [comment]
This was something that has been bothering me, I like this.

**Review by Pablete1234 — CHANGES_REQUESTED** [review]
The implementation of all the command stuff does not belong in PGM, it belongs in intake

**Comment by AustinLMayes** [comment]
`@DefaultCommand` is generic enough to exist in the base intake project itself.

**Comment by Electroid** [comment]
> `@DefaultCommand` is generic enough to exist in the base intake project itself.

Agreed, let's add this to `intake`.

**Pull request closed without merging**